playbooks/roles/dcnm_image_policy/dcnm_tests.yaml

```diff
@@ -25,6 +25,8 @@
     leaf2: 192.168.1.5
     leaf3: 192.168.1.6
     leaf4: 192.168.1.7
+    install_package_1: cfg_cmp-0.3.1.0-1.x86_64.rpm
```

Comment:
Here and elsewhere diffs don't look related to the VRF issues ... probably need to merge latest `develop` changes into this branch.

Reply:
Oops!  Sorry Mike.  Just merged.